pkg/hubble/metrics/policy/handler.go

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package policy
+
+import (
+	"context"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	"github.com/cilium/cilium/pkg/hubble/metrics/api"
+	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
+)
+
+type policyHandler struct {
+	verdicts *prometheus.CounterVec
+	context  *api.ContextOptions
+}
+
+func (d *policyHandler) Init(registry *prometheus.Registry, options api.Options) error {
+	c, err := api.ParseContextOptions(options)
+	if err != nil {
+		return err
+	}
+	d.context = c
+
+	labels := []string{"direction", "match"}
+	labels = append(labels, d.context.GetLabelNames()...)
+
+	d.verdicts = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: api.DefaultPrometheusNamespace,
+		Name:      "policy_verdicts_total",
+		Help:      "Total number of Cilium network policy verdicts",
+	}, labels)
+
+	registry.MustRegister(d.verdicts)
+	return nil
+}
+
+func (d *policyHandler) Status() string {
+	return d.context.Status()
+}
+
+func (d *policyHandler) ProcessFlow(ctx context.Context, flow *flowpb.Flow) {
+	if flow.GetEventType().GetType() != monitorAPI.MessageTypePolicyVerdict {
+		return
+	}
+	// ignore verdict if the source is host.
```

Comment:
Should we mention (and the ignored health) this in the help text? Might make it a bit too verbose, but might be useful to know.

Reply:
yeah let me document this 👍

Reply:
added a note in https://deploy-preview-20470--docs-cilium-io.netlify.app/operations/metrics.html#id13

> ``policy_verdicts_total`` metric does not count policy verdict events with ``reserved:host`` as the source since ``reserved:host`` is allowed to connect to any local endpoints regardless of whether a Cilium network policy rule explicitly allows it.